src/components/blocks/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ActivityCard from '../activityCard';
+import { StrapiActivity } from '../../../graphql-types';
+
+export interface ContentBlock {
+  title?: string;
+  text?: string;
+  background?: any;
+  activities?: Array<StrapiActivity>;
+  url?: string;
+  block_width: {
+    id?: number;
+    name?: string;
+    created_at?: string;
+    updated_at?: string;
+  };
+  image?: any;
+  strapi_component?: string;
+}
+
+const getWidth = (widthString: string | undefined) => {
+  switch (widthString) {
+    case '1/2':
+      return 'md:w-1/2';
+      break;
+    case '1/1':
+      return 'md:w-full';
+      break;
+    case '1/3':
+      return 'md:w-1/3';
+      break;
+    case '1/3':
+      return 'md:w-2/3';
+      break;
+    default:
+      return 'w-full';
+      break;
+  }
+};
+
+interface CommomnBlockProps {
+  block: ContentBlock;
+}
+
+export const GeneralBlock = ({ block }: CommomnBlockProps) => (
+  <div className={`${getWidth(block.block_width?.name)} flex-none inline-block mt-4 px-20 w-full`}>
+    {block.title && <h2>{block.title.toUpperCase()}</h2>}
+    {block.text && <div className="text-blue" dangerouslySetInnerHTML={{ __html: block.text }} />}
+    {block.image && <img className="w-full" src={block.image.url} />}
```

Comment:
This url doesn't work like this because it's relative url from Strapi, so it should be prepended with the Strapi server url.